src/Enterspeed.Source.UmbracoCms.V9/NotificationHandlers/BaseEnterspeedNotificationHandler.cs

```diff
@@ -18,24 +20,35 @@ public abstract class BaseEnterspeedNotificationHandler
         internal readonly IEnterspeedJobHandler _enterspeedJobHandler;
         internal readonly IUmbracoContextFactory _umbracoContextFactory;
         internal readonly IScopeProvider _scopeProvider;
+        internal readonly IEnterspeedJobFactory _enterspeedJobFactory;
```

Comment:
Is there any specific reason to have these dependencies injected? Doesn't seem to be used here.